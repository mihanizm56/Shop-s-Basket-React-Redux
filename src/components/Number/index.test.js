import React from "react"
import { shallow } from "enzyme"
import { Number } from "."

describe("Number", () => {
  const component = shallow(<Number />)

  it("Renders correct", () => {
    expect(component).toMatchSnapshot()
  })
})
