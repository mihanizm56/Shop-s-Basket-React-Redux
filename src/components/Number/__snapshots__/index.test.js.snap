// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Number Renders correct 1`] = `
<styled.span>
  8 (800) 500-75-55
</styled.span>
`;
